release-notes/opensearch-release-notes-2.2.0.md

```diff
@@ -0,0 +1,322 @@
+* Include new route to support update index while upload ([#34](https://github.com/opensearch-project/geospatial/pull/34))
+* Add uploader to upload user input ([#35](https://github.com/opensearch-project/geospatial/pull/35))
+* Make field name as optional ([#37](https://github.com/opensearch-project/geospatial/pull/37))
+* Use BulkResponse build error message ([#46](https://github.com/opensearch-project/geospatial/pull/46))
+* Update upload API response structure ([#51](https://github.com/opensearch-project/geospatial/pull/51))
+* Add metric and stat entity ([#54](https://github.com/opensearch-project/geospatial/pull/54))
+* Create Upload Stats Service to build response for stats API ([#62](https://github.com/opensearch-project/geospatial/pull/62))
+* Include stats api to provide upload metrics ([#64](https://github.com/opensearch-project/geospatial/pull/64))
+
+
+### OpenSearch Index Management
+* Ability to count the number of documents from source index ([#439](https://github.com/opensearch-project/index-management/pull/439)) and ([#3985](https://github.com/opensearch-project/OpenSearch/pull/3985))
+
+
+### OpenSearch k-NN
+* Lucene Based k-NN search support([#486](https://github.com/opensearch-project/k-NN/pull/486))
+
+
+### OpenSearch ML Commons
+* Add clustering function - RCFSummarize ([#355](https://github.com/opensearch-project/ml-commons/pull/355))
+* Add Logistic Regression algorithm ([#383](https://github.com/opensearch-project/ml-commons/pull/383))
+
+
+### OpenSearch SQL
+* Add match_phrase_prefix ([#661](https://github.com/opensearch-project/sql/pull/661))
+* `query_string` Relevance Function Implementation in SQL and PPL ([#675](https://github.com/opensearch-project/sql/pull/675))
+* Add Highlight In SQL ([#717](https://github.com/opensearch-project/sql/pull/717))
+
+
+## ENHANCEMENTS
+
+### OpenSearch Anomaly Detection
+* Make 1M1min possible ([#620](https://github.com/opensearch-project/anomaly-detection/pull/620))
+
+
+### OpenSearch Anomaly Detection Dashboards
+* Add feature attribution ([#296](https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/pull/296))
+
+
+### OpenSearch Index Management
+* Support mustache scripting of rollup.target_index field ([#435](https://github.com/opensearch-project/index-management/pull/435))
+* Support searching multiple rollup indices with same mapping ([#440](https://github.com/opensearch-project/index-management/pull/440))
+
+
+### OpenSearch k-NN
+* Add KNN codec that is based on Lucene92 codec([#444](https://github.com/opensearch-project/k-NN/pull/444))
+* Remove support for innerproduct for lucene engine([#488](https://github.com/opensearch-project/k-NN/pull/488))
+* Increase max dimension to 16k for nmslib and faiss([#490](https://github.com/opensearch-project/k-NN/pull/490))
+
+
+### OpenSearch Notifications
+* Adding K8s service name as webhook destination ([#455](https://github.com/opensearch-project/notifications/pull/455))
```

Comment:
Could we make the verbs at the beginning of each bullet point uniform?
So, instead of various versions ("Add", "Adds", "Adding", "Added") just have one unified version? "Adds" probably makes the most sense here.

Reply:
That would be amazing, along with standards for "Fix" vs. "Fixes" vs. "Fixed", and friends.

Reply:
This is the title of all the plugin repo PRs.